packages/odyssey-react-mui/src/labs/SideNav/OktaLogo.tsx

```diff
@@ -22,8 +22,8 @@ const OktaLogo = () => {
       xmlns="http://www.w3.org/2000/svg"
     >
       <path
-        fill-rule="evenodd"
-        clip-rule="evenodd"
+        fillRule="evenodd"
+        clipRule="evenodd"
```

Comment:
These are wrong, so I fixed them because I kept getting annoying `console.error` messages.

Reply:
I have this in my PR as well. Can you remove this in yours as I'm updating the OktaLogo anyways? It is not supposed to have both the `aura` and the `wordmark` anymore.

Reply:
Even if I merge this, it shouldn't cause issues right? And then if you merge your changes first, I'll get a merge conflict, so I can fix it on my end.